src/app/components/Cart/Cart.tsx

```diff
@@ -30,7 +36,45 @@ const styles = {
 };
 
 const Cart = () => {
-  const { items, clearCart } = useCartStore();
```

Comment:
to jest taki średni kawałek kodu - generalnie sobie po prostu o tym pogadamy w czwartek :)